### ISSUE TITLE
Can't download Fusionauth ebooks

### ISSUE THREAD

**Issue by sixhobbits** [FusionAuth/fusionauth-site#3340]
On the ebook pages such as https://fusionauth.io/ebooks/modern-guide-to-oauth and the others linked from the bottom, it is not possible to get access to the ebooks due to two (maybe unrelated) issues

## 1. Email not received 

After filling out the email address field and pressing submit, a confirmation message is seen, but an email is not received.

![screenshot 2024-10-22 at 14 36 25@2x](https://github.com/user-attachments/assets/c6b3fb65-fee9-4044-94fd-46ada1c6f537)

I have tried several different email addresses, across different domains and providers. I have checked spam and filtering. I have tried waiting a few hours. No emails were received.

## 2. Email box not fillable on Firefox

On Firefox, the email box is not fillable. Instead only a grey rectangle is seen.

<img width="1223" alt="image" src="https://github.com/user-attachments/assets/d802f36b-93d6-41b9-a84b-ebcc23ad13ec">

This might or might not be related to this recaptcha timeout 

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/14ae1eea-ad67-499b-8f5d-aa3bfa4a8a46">

## Details

Details on my versions of MacOS and Firefox are below. For the Firefox issue I have tried disabling add-ons, and using an incognito window, but the result is the same. 

<img width="233" alt="image" src="https://github.com/user-attachments/assets/f053e606-a682-48d0-ac47-9e561d37aa6e">

<img width="687" alt="image" src="https://github.com/user-attachments/assets/073f1046-d66a-447d-81dd-42faecaaff72">

**Comment by mooreds** [comment]
Did this get sorted out @sixhobbits ?

**Comment by sixhobbits** [comment]
@mooreds - I am still seeing similar behaviour to that described above. One of the 5 submissions I made before resulted in a received email. I tried a few more times now and got the ebook at one of the 5 email addresses I tried (different to last time). I tried 2x @companyname 2x @gmail and 1x @personaldomain emails. I got the email at one of the @companyname emails.

The firefox issue I narrowed down to 'enhanced tracing protection' being enabled. If I switch that off it loads as expected.

![screenshot 2024-12-13 at 10 04 41@2x](https://github.com/user-attachments/assets/d8778ce7-2541-45b9-b012-f00e72a79d0d)

I think email delivery rate still seems to be an issue and should probably be investigated.
